README.md

```diff
@@ -20,11 +20,18 @@ This module provides Stackdriver Debugger support for Node.js applications. [Sta
 # Install with `npm` or add to your `package.json`.
 npm install --save @google/cloud-debug
 
-# Require the agent at the top of your main script (but after '@google/cloud-trace' if you are also using it).
-require('@google/cloud-debug');
+# Require and start the agent at the top of your main script (but after '@google/cloud-trace' if you are also using it).
+require('@google/cloud-debug').start();
 ```
 Deploy your application, and navigate to the [Stackdriver Debug view][debug-tab] within the [Google Cloud Console][dev-console] to set snapshots and start debugging.
 
+## Configuration
+
+See [the default configuration](config.js) for a list of possible configuration options. These options can be passed to the agent through the object argument to the start command as shown below:
+```
+require('@google/cloud-debug').start({logLevel: 2});
```

Comment:
Other alternatives that might also make more compelling examples. `projectId` or `keyFilename`.